Code/Editor/2DViewport.cpp

```diff
@@ -952,7 +952,7 @@ void Q2DViewport::DrawViewerMarker(DisplayContext& dc)
     dc.SetColor(QColor(0, 0, 255)); // blue
     dc.DrawWireBox(-dim * noScale, dim * noScale);
 
-    float fov = GetIEditor()->GetSystem()->GetViewCamera().GetFov();
+    float fov = 60; // GetIEditor()->GetSystem()->GetViewCamera().GetFov();
```

Comment:
In general thanks to version control my rule of thumb is to delete all commented out code unless it's a specific part of documentation - please forgive the volume of "Please remove" comments here, they're all just reminders to mass clean the commented code.

To wit, please remove this comment (and maybe throw in a quick `constexpr float DefaultFov = 60.f;`.

Reply:
It was an intentional decision to leave much of the code commented out here. My rule of thumb is the same as yours, but in this case, I left it in for any future readers to see what used to happen there. Mainly because I didn't know if some of these code paths are reachable at all and didn't feel like trying to figure it out.

However it seems you're okay with going full steam ahead on removing all the legacy cry code, in which case I'll delete all the commented code.

Reply:
Addressed in 70b430082f